plc4j/spi/src/main/java/org/apache/plc4x/java/spi/messages/DefaultPlcReadRequest.java

```diff
@@ -139,6 +136,19 @@ public PlcReadRequest build() {
             return new DefaultPlcReadRequest(reader, parsedTags);
         }
 
+        public PlcTag parseOrNull(String tagAddress){
```

Comment:
Generally the same comment as for the previous, in addition I personally don't really like the "or-null" variant ... I would prefer to keep only the optional one.